src/esp/assets/BaseMesh.cpp

```diff
@@ -95,24 +130,28 @@ void BaseMesh::buildSemanticOBBs(
     // semantic ID on vertex - valid values are 1->semanticIDToSSOBJidx.size().
     // Invalid/unknown semantic ids are > semanticIDToSSOBJidx.size()
     const auto semanticID = vertSemanticIDs[vertIdx];
-    if ((semanticID > 0) && (semanticID < semanticIDToSSOBJidx.size())) {
+    if ((semanticID >= 0) && (semanticID < semanticIDToSSOBJidx.size())) {
```

Comment:
Isn't the semanticId always greater than or equal to 0 in practice? I seem to remember there being an implicit uint -> int conversion somewhere.

Reply:
Oh nvm, I see, empty values are initialized to -1 above even though the semantic category can't return them.